packages/cli/src/Server.ts

```diff
@@ -638,6 +663,25 @@ class App {
 				}
 			}
 
+			if (tags) {
+				await TagHelpers.deleteAllTagsForWorkflow(id);
+
+				const tagIds = tags.split(',');
+
+				for (const tagId of tagIds) {
+					await TagHelpers.validateId(tagId);
+				}
+
+				await TagHelpers.createRelations(id, tagIds);
+
+				const foundTags = await Db.collections.Tag!.find({
+					select: ['id', 'name'],
+					where: { id: In(tagIds) },
+				});
+				responseData.tags = foundTags.map(({ id, name }) => ({ id: id.toString(), name }));
```

Comment:
can this be a utility function? since it's duplicated logic with adding

Reply:
They are not identical: the create version (as refactored now) does not call `validateNotRelated` and `removeRelations`. That said, the two versions can be consolidated in a single function with a flag for whether those two should be called. Is this advisable?

Reply:
I see. thanks. the current implementation you have is better.